numpy/core/src/umath/override.c

```diff
@@ -180,18 +179,6 @@ normalize_reduceat_args(PyUFuncObject *ufunc, PyObject *args,
 }
 
 static int
-normalize_outer_args(PyUFuncObject *ufunc, PyObject *args,
-                    PyObject **normal_args, PyObject **normal_kwds)
-{
```

Comment:
I might have been tempted to keep this as a simple wrapper that forwards to `__call__`, with a brief comment explaining they are the same. The compiler should inline the single function call anyway, right?